### ISSUE TITLE
There's no prompt when you hit enter

### ISSUE THREAD

**Issue by davidwdan** [clue/php-readline-react#1]
I'm running into an issue where when I press enter, the prompt does not appear.  I'm not sure if this is a OSX related issue or what, but adding the following lines at line 123 fixes the issue:

```PHP
readline_callback_handler_remove();
readline_callback_handler_install($this->prompt, array($this, 'handleLine'));
```

**Comment by mbonneau** [comment]
I have tested it on in the following environment with the same results:

```
[ examples]$ cat /etc/redhat-release 
CentOS Linux release 7.0.1406 (Core) 
[ examples]$ php -v
PHP 5.6.14 (cli) (built: Oct 16 2015 07:11:57) 
Copyright (c) 1997-2015 The PHP Group
Zend Engine v2.6.0, Copyright (c) 1998-2015 Zend Technologies
    with Zend OPcache v7.0.6-dev, Copyright (c) 1999-2015, by Zend Technologies
```

**Comment by clue** [comment]
Thanks for spotting!

Looks like even the docs suggest one should simply call `readline_callback_handler_install()` again in order to redraw the prompt: http://php.net/manual/en/function.readline-callback-handler-install.php

Does anybody feel like filing a PR? :+1: 

FWIW: This project was a mere experiment and I stopped using this after realizing its limitations. I would encourage everybody to use [clue/stdio-react](https://github.com/clue/php-stdio-react) instead.

**Comment by davidwdan** [comment]
@clue looks like stdio-react doesn't support autocomplete and history yet, so I'll keep using this project for the time being.

**Comment by clue** [comment]
Thanks for your PR! You're right, stdio-react is still under active development, see https://github.com/clue/php-stdio-react/issues/1 and https://github.com/clue/php-stdio-react/issues/2. Make sure to watch the repo or subscribe to the issues so you'll get notified once they're in :)

**Comment by clue** [comment]
> @clue looks like stdio-react doesn't support autocomplete and history yet, so I'll keep using this project for the time being.

A few days ago I've released a v1.0.0 which supports both autocompletion and history support :shipit:

**Comment by clue** [comment]
Closed via #2
